jade_job/main.py

```diff
@@ -76,7 +75,7 @@ def is_done(job_id: str, env: str) -> bool:
     # if "succeeded" then we want to stop polling, so true
     # if "failed" then we want to stop polling, so true
     status = check_job_status(job_id, env)
-    return status in ['succeeded', 'failed']
+    return status in {'succeeded', 'failed'}
```

Comment:
does the `{ }` notation make it into a set?

Reply:
yup! it doesn't really make a difference in this small list, just my OCD to use sets for existence check ;P